docs/getting-started.md

```diff
@@ -1,22 +1,40 @@
 Getting Started
 ===
 
-## What is DeepOps?
+## Requirements
 
-The DeepOps project is a reference approach for provisioning and deploying GPU servers and multi-node GPU clusters. It is intended to be adapted to unique environments and requirements while helping you go faster by sharing the experience we have with GPU servers and NVIDIA SW stacks applied to our HPC and DL needs.
+* A pre-existing "provisioning" node which can be used to run Ansible and the install scripts
+* A cluster to deploy to (potentially a cluster of one server or a [virtual one](/virtual/README.md))
```

Comment:
Can we say something like 'cluster or single server'? 'cluster of one server' doesn't make sense

Reply:
Done